src/AdminService.Data/ServiceEndpointRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.AdminService.Data.Interfaces;
+using LT.DigitalOffice.AdminService.Data.Provider;
+using LT.DigitalOffice.AdminService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.AdminService.Data
+{
+  public class ServiceEndpointRepository : IServiceEndpointRepository
+  {
+    private readonly IDataProvider _provider;
+
+    public ServiceEndpointRepository(
+      IDataProvider provider)
+    {
+      _provider = provider;
+    }
+
+    public async Task<bool> CreateAsync(List<DbServiceEndpoint> servicesEndpoints)
+    {
+      if (!servicesEndpoints.Any())
```

Comment:
if servicesEndpoints will be null servicesEndpoints.Any() will throw an exception